projects/swimlane/ngx-ui/src/lib/components/date-range-calendar/date-range-picker.component.html

```diff
@@ -0,0 +1,95 @@
+<ngx-dropdown [closeOnClick]="false" [closeOnOutsideClick]="true" #wrapperRef>
+  <ngx-dropdown-menu class="ngx-dropdown-menu ngx-dropdown-dark-outline" (click)="$event.stopPropagation()">
+    <div class="date-range-selection"><span>Date range selection </span></div>
+    <div class="date-range-picker">
+      <!-- Sidebar Presets -->
+      <div class="sidebar">
+        <p
+          *ngFor="let preset of presets"
+          [class.active]="selectedPreset === preset.label"
+          (click)="selectPreset(preset)"
+        >
+          {{ preset.label }}
+        </p>
+      </div>
```

Comment:
Top and bottom margin of the list should match the mockups and/or be equal, 0 according to the mockup. The first item has some margin while the user can scroll and not observe margin on the last item.

<img width="245" alt="Screenshot 2025-07-08 at 7 23 18 AM" src="https://github.com/user-attachments/assets/860572a1-6bdc-498e-a43a-e55dcc206de8" />
<img width="384" alt="Screenshot 2025-07-08 at 7 23 04 AM" src="https://github.com/user-attachments/assets/a5a7e81d-442b-458e-8ef0-7df68557e1eb" />